commodore/component/compile.py

```diff
@@ -25,6 +25,8 @@ def compile_component(config: Config, component_path, value_files, search_paths,
     component_path = P(component_path).resolve()
     value_files = [P(f).resolve() for f in value_files]
     search_paths = [P(d).resolve() for d in search_paths]
+    search_paths.append('./dependencies/')
+    search_paths.append(os.path.join(component_path, 'vendor'))
```

Comment:
```suggestion
    search_paths.append(component_path / 'vendor')
```

Reply:
looks like the suggestion was not fully applied